tests/UIButtons.js

```diff
@@ -0,0 +1,173 @@
+    await browser.pause(5000);
+  });
+
+  it("Checking the appearance of the menu buttons", async () => {
+    const buttons = [
+      Menu.cuteButton,
+      Menu.funnyButton,
+      Menu.goodButton,
+      Menu.omgButton,
+      Menu.moreButton,
+    ];
+
+    for (let i = 0; i < buttons.length; i++) {
+      const elem = buttons[i];
+
+      const fontSize = await elem.getCSSProperty("font-size");
+      console.log(fontSize);
+      const valueFontSize = fontSize.value;
+      await expect(valueFontSize).toEqual("16px");
+
+      const minWidth = await elem.getCSSProperty("min-width");
+      console.log(minWidth);
+      const valueMinWidth = minWidth.value;
+      await expect(valueMinWidth).toEqual("120px");
+
+      const borderLeft = await elem.getCSSProperty("border-left-width");
+      console.log(borderLeft);
+      const valueBorderLeft = await borderLeft.value.slice(0, -2);
+      await expect(Math.round(valueBorderLeft)).toEqual(2);
+
+      const borderTop = await elem.getCSSProperty("border-top-width");
+      console.log(borderTop);
+      const valueBorderTop = await borderTop.value.slice(0, -2);
+      await expect(Math.round(valueBorderTop)).toEqual(5);
+
+      const color = await elem.getCSSProperty("color");
+      console.log(color);
+      const valueColor = color.parsed.hex;
+      await expect(valueColor).toEqual("#898989");
+
+      const alignItems = await elem.getCSSProperty("align-items");
+      console.log(alignItems);
+      const valueAlignItems = alignItems.value;
+      await expect(valueAlignItems).toEqual("center");
+    }
+  });
+
+  it("Checking the appearance of the submenu buttons", async () => {
+    const submenuButtons = [
+      await Menu.aboutButton,
+      Menu.privacyPolicyButton,
+      Menu.dmcaButton,
+      Menu.advertisingButton,
+    ];
+
+    await (await Menu.moreButton).waitForDisplayed();
+    await (await Menu.moreButton).click();
+    await browser.pause(5000);
+
+    for (let i = 0; i < submenuButtons.length; i++) {
+      const object = await submenuButtons[i];
+
+      await browser.pause(5000);
+
+      let fontSize = await object.getCSSProperty("font-size");
+      console.log(fontSize);
+      const valueFontSize = await fontSize.value;
+      await expect(valueFontSize).toEqual("16px");
+
+      const color = await object.getCSSProperty("color");
+      console.log(color);
+      const valueColor = await color.parsed.hex;
+      await expect(valueColor).toEqual("#898989");
+
+      const pandding = await object.getCSSProperty("padding-right");
+      console.log(pandding);
+      const valuePandding = await pandding.value;
+      await expect(valuePandding).toEqual("10px");
+
+      const width = await object.getCSSProperty("width");
+      console.log(width);
+      const valueWidth = await width.value;
+      await expect(valueWidth).toEqual("150px");
+    }
+  });
+
+  it("Checking the appearance of the search button", async () => {
```

Comment:
Are you referring here to the _search input_? Or which search button? That search text is called placeholder for the search input :)
![image](https://user-images.githubusercontent.com/71023603/137342753-e26aa5bf-66e6-4125-afc1-1279d860e43d.png)